### ISSUE TITLE
Add Markdown support

### ISSUE THREAD

**Issue by vendruscolo** [memberful/memberful-wp#184]
It'd be great for us being able to write the _marketing content_ in Markdown, at this time only HTML is supported.

Many sites use Jetpack to get Markdown support (we do), can you detect it and use its Markdown converter (to get consistent output with the main editor)?

**Comment by dstrojny** [comment]
Good idea @vendruscolo! We'll take a look at supporting this in our next update (if it is possible).

**Comment by vendruscolo** [comment]
It should be possibile — I've just found this (I was looking about how to add MD support to custom post types) http://www.blogercise.com/add-jetpack-markdown-support-to-plugins/
